pnpm-lock.yaml

```diff
@@ -33958,7 +33960,7 @@ packages:
     resolution: {integrity: sha512-qo3glqyTa61Ytg4u73GultjHGjdRyig3tG6lPtyX/jOEJvHif9uB0/OCI2Kif6ctF3caQTW2G5gym21oAsI4pw==}
     engines: {node: '>= 10.13.0'}
     peerDependencies:
-      webpack: ^4.0.0 || ^5.0.0 || 5.83.0
+      webpack: ^4.0.0 || ^5.0.0 || ^5.82.0
```

Comment:
these changes seem odd to me.

I'm unclear why our lock file is in the state that its in, it seems that https://github.com/microsoft/FluidFramework/pull/15607/files#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bb made massive change to the lock file for client when changing server?

Reply:
This has been resolved outside of this PR, and the lockfile diffs should now be smaller and correct.